yb-voyager/src/callhome/diagnostics.go

```diff
@@ -275,8 +275,9 @@ Version History:
 1.0: Added fields for BatchSize, OnPrimaryKeyConflictAction, EnableYBAdaptiveParallelism, AdaptiveParallelismMax
 1.1: Added YBClusterMetrics field, and corresponding struct - YBClusterMetrics, NodeMetric
 1.2: Split out the data metrics into a separate struct - ImportDataMetrics
+1.3: Added CurrentParallelConnections field to ImportDataMetrics
 */
-var IMPORT_DATA_CALLHOME_PAYLOAD_VERSION = "1.2"
```

Comment:
we don't have the payload version with import data file phase payload should we add ?

Reply:
Oh I just realise, we don't update the phase payload of import data file for common things of `import data` and `import data file`

Reply:
We can add a TODO to fix that later in a separate task.